src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/RestartRequestBar.xaml.cs

```diff
@@ -37,7 +36,7 @@ public partial class RestartRequestBar : UserControl, INuGetProjectContext
 
         public NuGetActionType ActionType { get; set; }
 
-        public INuGetTelemetryService TelemetryService { get; set; }
+        public KeyValuePair<string, Guid> OperationId { get; set; }
```

Comment:
Should this be a NuGet class instead of KeyValuePair? Since this is used all over it seems difficult to keep context on what this is.

Reply:
Discussed offline, this is fine